qa/suites/upgrade/client-upgrade-mimic/mimic-client-x/basic/2-workload/rbd_api_tests.yaml

```diff
@@ -0,0 +1,21 @@
+tasks:
```

Comment:
Client upgrades no longer function since the C++ ABI is not backwards compatible.

Reply:
Wow !  So no need in this suite, or we need to do something else @dillaman ?

Reply:
We could copy a QEMU or the OpenStack devstack test to ensure that the new librbd API still supports those applications.We also need to test luminous cluster against nautilus clients (just had a breaking bug discovered yesterday that prevents nautilus librbd from working with luminous OSDs).

Reply:
I've added `nautilus` to the cron, pls confirm that this is what you meant:

50 01 * * *  CEPH_BRANCH=nautilus; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH  -n 7 -m $MACHINE_NAME -s upgrade/client-upgrade-luminous -k distro -e $CEPH_QA_EMAIL --suite-branch luminous

Reply:
Pls point to "QEMU or the OpenStack devstack test"

Reply:
@yuriw I am suggesting that we test with the OSDs at both Luminous and Mimic; and clients at Nautilus. The OpenStack devstack example is here [1] -- just run the client workload on its own node so that the client software can be upgraded to Nautilus.

[1] https://github.com/ceph/ceph/blob/master/qa/suites/rbd/openstack/workloads/devstack-tempest-gate.yaml

Reply:
@dillaman This is what you meant http://tracker.ceph.com/issues/38991 ?

(I will cycle back on this suite soon.)

Reply:
Yeah -- that's a missing C++ API symbol.